avm/src/daemon/linux.rs

```diff
@@ -39,71 +39,99 @@ struct VMStatsV1 {
   freeSwapKb: u64,
 }
 
-pub async fn post(endpoint: &str, body: Value) -> String {
-  let resp = post_v1(endpoint, body).await;
+async fn get_private_ip() -> String {
+  let res = Command::new("hostname")
+    .arg("-I")
+    .output()
+    .await;
+  let err = "Failed to execute `hostname`";
+  let stdout = res.expect(err).stdout;
+  String::from_utf8(stdout)
+    .expect(err)
+    .trim()
+    .to_string()
+}
```

Comment:
Are you using this function?

Reply:
You are, you not only want to trim it, but split on `\n` and take just the first value, just in case.

Reply:
Wait, sorry, split on space ` `, not `\n`